lib/main.rb

```diff
@@ -0,0 +1,17 @@
+def bubble_sort(numbers)
+  sorted = true
+
+  (numbers.length - 1).times do |i|
+    sub_arr = [numbers[i], numbers[i + 1]]
+
+    next if sub_arr[0] <= sub_arr[1]
+
+    numbers = numbers[0, i] + sub_arr.sort! + numbers[i + 2...]
```

Comment:
using `sort!` in a sorting algorithm must be cheating 🤣. Why not something like this?
```suggestion
    numbers[i] = sub_arr[1]
    numbers[i + 1] = sub_arr[0]
```

Reply:
jajajajjajajajjajajaja yeah, better! ajjajjajajajajjaajjajjaja